pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -32,6 +33,37 @@ import (
 	"time"
 )
 
+func createTestInferenceService(serviceKey types.NamespacedName, hasStorageUri bool) *v1beta1.InferenceService {
+
+	predictor := v1beta1.PredictorExtensionSpec{
```

Comment:
Consider inlining this and adding the if logic at the end of the function.

Reply:
Wouldn't creating InferenceService according to parameters cleaner than copying pasting InferenceService spec?